eZ/Publish/Core/Repository/Permission/PermissionResolver.php

```diff
@@ -405,7 +405,7 @@ private function isDeniedByRoleLimitation(
      *         }
      *     );
      *
-     * @param \Closure $callback
+     * @param \callable(\eZ\Publish\API\Repository\Repository): mixed $callback
```

Comment:
raising internal question here: why does the definition of this parameter differ from `Repository::sudo`?

Reply:
There seems to be some sort of a difference between declaring types like this for an interface and for an actual implementation (or at least, when it's interface-less).

See https://github.com/phpstan/phpstan/issues/4913 / https://phpstan.org/r/370e514d-13da-45c2-b735-478143d91976.